frontend/selectors/index.jsx

```diff
@@ -0,0 +1,76 @@
+import { getProduct } from '@shopgate/engage/product';
+import { createSelector } from 'reselect';
+import { referenceUnitProp } from '../config';
+import { calculateBasePrice, calculateBaseUnit } from '../helpers';
+
+export const getHasTierPrice = createSelector(
+  getProduct,
+  (productData) => {
+    if (!productData || productData.isFetching) {
+      return false;
+    }
+
+    if (productData.price.tiers && productData.price.tiers.length) {
+      return true;
+    }
+
+    return false;
+  }
+);
+
+export const getHasStrikePrice = createSelector(
+  getProduct,
+  (productData) => {
+    if (!productData || productData.isFetching) {
+      return false;
+    }
+
+    if (productData.price.unitPriceStriked && productData.price.unitPriceStriked > 0) {
+      return true;
+    }
+
+    return false;
+  }
+);
+
+export const getReferenceUnit = createSelector(
+  getProduct,
+  (productData) => {
+    if (!productData || productData.isFetching) {
+      return null;
+    }
+
+    const referenceUnit = productData.additionalProperties
```

Comment:
If the product-properties extension is not deployed, this line will throw an error: Uncaught TypeError: Cannot read properties of undefined (reading 'find')